gradle/libs.versions.toml

```diff
@@ -84,6 +85,7 @@ ktlint = { id = "org.jlleitschuh.gradle.ktlint", version.ref = "ktlint" }
 activity-compose = { module = "androidx.activity:activity-compose", version.ref = "activity-compose" }
 activity-ktx = { module = "androidx.activity:activity-ktx", version.ref = "activity-compose" }
 android-beacon-library = { module = "org.altbeacon:android-beacon-library", version.ref = "androidBeaconLibrary" }
+androidx-connect-client = { module = "androidx.health.connect:connect-client", version.ref = "connectClient" }
```

Comment:
Maybe name this `androidx-health-connect-client` to make it more recognizable outside this file?